src/canopy_calcs.F90

```diff
@@ -209,123 +219,168 @@ SUBROUTINE canopy_calcs
                                 end if
                             end if
 
+!.......user option to calculate in-canopy leafage influence and assigning LAI as per timestep
+
+
+                            if (leafage_opt .eq. 0) then
+                                ! Initialize pastlai and currentlai based on current timestep
```

Comment:
OK, so if leafage is off, then these variables are not initialized and not used right?

Reply:
Yes